tools/rebase-onto-formatter.sh

```diff
@@ -41,17 +41,56 @@ fi
 
 echo "Running the script now. This might take a while..."
 
-# The idea behind the sscript is ... TODO
+# The general idea is the following:
+#
+# We have a branch consisting of changes C1, C2, ... on top of our BASE_COMMIT 0.
+# Also, from the base commit, a formatting change `f(0)` was made.
```

Comment:
```suggestion
# Also, from the base commit, a formatting change `f(0)` was made on some base branch (eg., develop, or master).
```

this would have helped me to click slightly faster.

Reply:
thanks.